src/Controls/src/Core/DragAndDrop/DragStartingEventArgs.cs

```diff
@@ -21,21 +21,29 @@ public DragStartingEventArgs()
 		/// Initializes a new instance of the <see cref="DragStartingEventArgs"/> class.
 		/// </summary>
 		/// <param name="getPosition">Function used to get the position relative a specified <see cref="IElement"/>.</param>
-		internal DragStartingEventArgs(Func<IElement?, Point?>? getPosition)
+		/// <param name="platformArgs">The platform-specific data associated with the drag.</param>
+		public DragStartingEventArgs(Func<IElement?, Point?>? getPosition, PlatformDragStartingEventArgs? platformArgs)
```

Comment:
I see this is public now, some of the others are and some are not? Still in progress or?

Reply:
All the EventArgs that are not platformEventArgs should have public constructors now but let me know if I am missing one?